tests/legacy/test_monsieurcuisine.py

```diff
@@ -0,0 +1,105 @@
+# mypy: allow-untyped-defs
+import responses
+
+from recipe_scrapers.monsieurcuisine import MonsieurCuisine
+from tests.legacy import ScraperTest
+
+
+class TestMonsieurCuisineScraper(ScraperTest):
+    scraper_class = MonsieurCuisine
+
+    @classmethod
+    def expected_requests(cls):
+        yield responses.GET, "https://www.monsieur-cuisine.com/de/recipe/paprika-nudel-auflauf-mit-hackfleisch", "tests/legacy/test_data/monsieurcuisine.testhtml"
+        expected_headers = {"Accept-Language": "de-DE", "Device-Type": "web"}
+        yield responses.GET, "https://mc-api.tecpal.com/api/v2/recipes/1741327", "tests/legacy/test_data/monsieurcuisine.testjson", expected_headers
+
+    def test_host(self):
+        self.assertEqual("monsieur-cuisine.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Monsieur Cuisine Official", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual(
+            "Paprika-Nudel-Auflauf mit Hackfleisch", self.harvester_class.title()
+        )
+
+    def test_prep_time(self):
+        self.assertEqual(70, self.harvester_class.prep_time())
+
+    def test_cook_time(self):
+        self.assertEqual(0, self.harvester_class.cook_time())
```

Comment:
Something seems to be wrong here; this isn't an instant / no-cook-time recipe.

Reply:
This is as the recipe is posted. It states that the prep time is the whole time it takes for the recipe.

Reply:
Ok, thanks for checking that :+1: